src/main/scala/io/opentargets/etl/ETLStep.scala

```diff
@@ -0,0 +1,24 @@
+package io.opentargets.etl
+
+import com.typesafe.scalalogging.LazyLogging
```

Comment:
Should this be in the PR @mkarmona? It looks like something that you're still working on? Same with the changes in `Main.scala`